playbooks/roles/ocp-upgrade/tasks/main.yaml

```diff
@@ -1,22 +1,16 @@
 ---
 
-- name: Fetch cluster operators details
-  shell: "oc get co"
-  register: oplog
-  until: "'AVAILABLE' in oplog.stdout"
-  retries: 3
-  delay: 3
-
-- name: Process cluster operators details
-  shell: |
-    echo "{{ oplog.stdout }}" | awk '/PROGRESSING/,EOF {print $4}' | grep 'True' | wc -l
-    echo "{{ oplog.stdout }}" | awk '/DEGRADED/,EOF {print $5}' | grep 'True' | wc -l
-  register: opcount
+- name: Retry till all co are Available/NOT PROGRESSING/NOT DEGRADED state
```

Comment:
Can we have either blocks or title format for all?
```
Available/NOT Progressing/NOT Degraded
```

Reply:
Updated title to 'Check all co are in 'Available' state and not in 'Progressing' or 'Degraded' state'